index.html

```diff
@@ -433,6 +443,19 @@ <h2><dfn>disconnect()</dfn> method</h2>
         <a data-cite="!WHATWG-DOM#context-object">context object</a>'s
         <a>observer buffer</a>.</p>
       </section>
+      <section>
+        <h2><dfn>supportedEntryTypes</dfn> attribute</h2>
+        <p>The user agent must maintain <dfn>supported entry types</dfn>, a list
+        of strings representing the entry types which the user agent supports
+        for the <a>PerformanceObserver</a> interface. This list is populated by
+        specs that define new entry types via the algorithm to
```

Comment:
Nits:
s/specs/specifications/
s/via the algorithm to register performance entry type/via the register performance entry type algorithm/